RadClass/RadClass.py

```diff
@@ -220,4 +220,7 @@ def write(self, filename):
         Write results to file using Pandas' to_csv() method.
         filename should include the file extension.
         '''
+        self.storage = pd.DataFrame.from_dict(self.storage,
+                                              orient='index',
+                                              columns=np.arange(len(self.cache[0])))
```

Comment:
This line adds labels to the `DataFrame` and subsequent CSV file. It is optional, and may not be optimal, but this was the fastest/safest way to find the number of energy bins I could think of.